web/styles/settings.css

```diff
@@ -1286,7 +1284,7 @@ label.preferences-radio-choice-label {
 #settings_page {
     height: 95vh;
     width: 97vw;
-    max-width: 1024px;
+    max-width: 73.1428em; /* 1024px at 14px em */
```

Comment:
I'd round this up to `73.1429em` (since it works out to 73.14285...)

Reply:
Sure